### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by dtoncu**
Fixes typos from docstrings, comments and logging messages.

### CODE REVIEW

File: argus/tests/cloud/windows/test_smoke.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def _parse_licenses(output):
 
 
 class TestSmoke(smoke.TestsBaseSmoke):
-    """Test additional Windows specific behaviour."""
```
Comment (stefan-caraiman) [inline argus/tests/cloud/windows/test_smoke.py:51]: This is a spelling difference(between American and British english) not a typo.

File: argus/introspection/cloud/base.py [hunk 2]
```diff
@@ -22,19 +22,19 @@
...
         """Return the disk size from the instance."""
 
     @abc.abstractmethod
     def username_exists(self, username):
-        """Check if the given username exists in the instance."""
+        """Check if the given user-name exists in the instance."""
 
     @abc.abstractmethod
     def get_instance_hostname(self):
-        """Get the hostname of the instance."""
```
Comment (stefan-caraiman) [inline argus/introspection/cloud/base.py:37]: `hostname` is the right form.

File: argus/introspection/cloud/base.py [hunk 3]
```diff
@@ -22,19 +22,19 @@
...
-    """Introspection class for testing cloudbase-init."""
+    """Introspection class for testing Cloudbase-Init."""
 
     @abc.abstractmethod
     def get_disk_size(self):
         """Return the disk size from the instance."""
 
     @abc.abstractmethod
     def username_exists(self, username):
-        """Check if the given username exists in the instance."""
```
Comment (stefan-caraiman) [inline argus/introspection/cloud/base.py:33]: `username` is the right form.

File: argus/recipes/cloud/windows.py [hunk 4]
```diff
@@ -211,19 +212,19 @@ def wait_cbinit_finalization(self):
...
-        """Prepare the cloudbase-init config."""
+        """Prepare the Cloudbase-Init config."""
         self._cbinit_conf = cbinit_config.CBInitConfig(
             client=self._backend.remote_client)
 
         self._cbinit_unattend_conf = cbinit_config.UnattendCBInitConfig(
             client=self._backend.remote_client)
 
         # NOTE(mmicu): Because first_logon_behaviour will control
-        #              how the password should work on next logon,
```
Comment (stefan-caraiman) [inline argus/recipes/cloud/windows.py:231]: `logon` is fine.

File: argus/recipes/cloud/windows.py [hunk 5]
```diff
@@ -301,7 +302,7 @@ def pre_sysprep(self):
 
 
 class BaseNextLogonRecipe(CloudbaseinitRecipe):
-    """Useful for testing the next logon behaviour."""
```
Comment (stefan-caraiman) [inline argus/recipes/cloud/windows.py:309]: Ditto

File: argus/recipes/cloud/windows.py [hunk 6]
```diff
@@ -313,19 +314,19 @@ def prepare_cbinit_config(self, service_type):
 
 
 class AlwaysChangeLogonPasswordRecipe(BaseNextLogonRecipe):
-    """Always change the password at next logon."""
```
Comment (stefan-caraiman) [inline argus/recipes/cloud/windows.py:321]: Ditto

File: argus/recipes/cloud/windows.py [hunk 7]
```diff
@@ -313,19 +314,19 @@ def prepare_cbinit_config(self, service_type):
...
+    """Always change the password at next log on."""
 
     behaviour = 'always'
 
 
 class ClearPasswordLogonRecipe(BaseNextLogonRecipe):
-    """Change the password at next logon if the password is from metadata."""
+    """Change the password at next log on if the password is from metadata."""
```
Comment (stefan-caraiman) [inline argus/recipes/cloud/windows.py]: Ditto

File: argus/recipes/cloud/windows.py [hunk 8]
```diff
@@ -436,7 +437,7 @@ def prepare_cbinit_config(self, service_type):
 
 
 class CloudbaseinitLongHostname(CloudbaseinitRecipe):
-    """Recipe for testing the netbios long hostname compatibility option."""
```
Comment (stefan-caraiman) [inline argus/recipes/cloud/windows.py:444]: `hostname` is the right form.

File: argus/tests/cloud/smoke.py [hunk 9]
```diff
@@ -234,10 +234,10 @@ def test_set_timezone(self):
 
 
 class TestSetHostname(base.BaseTestCase):
-    """Test that the expected hostname was set in the instance."""
+    """Test that the expected host-name was set in the instance."""
```
Comment (stefan-caraiman) [inline argus/tests/cloud/smoke.py]: Ditto

File: argus/tests/cloud/smoke.py [hunk 10]
```diff
@@ -234,10 +234,10 @@ def test_set_timezone(self):
...
 
     def test_set_hostname(self):
-        # Verify that the instance hostname matches what we are
+        # Verify that the instance host-name matches what we are
```
Comment (stefan-caraiman) [inline argus/tests/cloud/smoke.py]: Ditto

File: argus/tests/cloud/smoke.py [hunk 11]
```diff
@@ -247,7 +247,7 @@ def test_set_hostname(self):
 class TestLongHostname(base.BaseTestCase):
 
     def test_hostname_compatibility(self):
-        # Verify that the hostname is set accordingly
```
Comment (stefan-caraiman) [inline argus/tests/cloud/smoke.py:250]: Ditto

File: argus/tests/cloud/smoke.py [hunk 12]
```diff
@@ -300,7 +300,7 @@ def test_disk_expanded(self):
         self.assertGreater(disk_size, datastore_size)
 
     def test_hostname_set(self):
-        # Test that the hostname was properly set.
```
Comment (stefan-caraiman) [inline argus/tests/cloud/smoke.py:303]: Ditto

File: argus/action_manager/windows.py [hunk 13]
```diff
@@ -210,7 +209,7 @@ def sysprep(self):
                 winrm_exceptions.InvalidCredentialsError,
                 requests.ConnectionError, requests.Timeout):
             # After executing sysprep.ps1 the instance will reboot and
-            # it is normal to have conectivity issues during that time.
+            # it is normal to have connectivity issues during that time.
             # Knowing this we have to except this kind of errors.
```
Comment (stefan-caraiman) [inline argus/action_manager/windows.py]: these kind*

File: argus/scenarios/base.py [hunk 14]
```diff
@@ -122,14 +122,14 @@ class BaseScenario(unittest.TestCase):
...
-        This means that the backend will be instantiated and an
+        This means that the back-end will be instantiated and an
         instance will be created and prepared. After the preparation
         is finished, the tests can run and can introspect the instance
         to check what they are supposed to be checking.
         """
         # pylint: disable=not-callable
-        # Pylint is not aware that the attrs are reassigned in other modules,
-        # so we're just disabling the errors for now.
+        # Pylint is not aware that the modules, are reassigned in other
```
Comment (stefan-caraiman) [inline argus/scenarios/base.py]: leave it like `attrs`

File: argus/scenarios/base.py [hunk 15]
```diff
@@ -159,15 +159,16 @@ def setUpClass(cls):
     def prepare_instance(cls):
         """Prepare the underlying instance."""
         # pylint: disable=not-callable
-        # Pylint is not aware that the attrs are reassigned in other modules,
-        # so we're just disabling the errors for now.
+        # Pylint is not aware that the attributes are reassigned in other
```
Comment (stefan-caraiman) [inline argus/scenarios/base.py]: Ditto

**Review by stefan-caraiman — COMMENTED** [review]
Also please rewrite the commit message like so:

`Fix typos

Fixes typos from docstrings, comments and logging messages.`

File: argus/resources/windows/sysprep.ps1 [hunk 16]
```diff
@@ -10,7 +10,7 @@ try
 
     if (-Not (Test-Path -LiteralPath $unattendedXmlPath -PathType Leaf)){
     # if there is no Unnatended.xml the command will halt and we will never
```
Comment (ader1990) [inline argus/resources/windows/sysprep.ps1]: Unattend.xml

File: argus/resources/windows/installCBinit.ps1 [hunk 17]
```diff
@@ -8,7 +8,7 @@ $ErrorActionPreference = "Stop"
 
 
 function Set-CloudbaseInitServiceStartupPolicy {
-    #Cloudbase Init service must start only after the sysprep has rebooted the
```
Comment (ader1990) [inline argus/resources/windows/installCBinit.ps1:11]: spaces after #

**Review by ader1990 — CHANGES_REQUESTED** [review]

File: argus/config_generator/windows/cb_init.py [hunk 18]
```diff
@@ -68,26 +68,26 @@ def _config_specific_paths(self):
...
 
     def set_service_type(self, service_type):
         """Set the service type config.
 
         :param service_type:
-            This can be a string like http, configdrive, ec2, opennebula,
-            cloudstack, MAAS or it can be a list with the required service
+            This can be a string like HTTP, ConfigDrive, EC2, OpenNebula,
+            CloudStack, MAAS or it can be a list with the required service
             types. The order is important.!
```
Comment (stefan-caraiman) [inline argus/config_generator/windows/cb_init.py:85]: `.!` don't think you need that dot

**Review by ader1990 — APPROVED** [review]

File: argus/scenarios/base.py [hunk 19]
```diff
@@ -162,12 +162,13 @@ def prepare_instance(cls):
         # Pylint is not aware that the attrs are reassigned in other modules,
         # so we're just disabling the errors for now.
         cls.recipe = cls.recipe_type(cls.backend)
+
```
Comment (stefan-caraiman) [inline argus/scenarios/base.py:165]: is this blank line required?

**Pull request merged**